fe/fe-core/src/main/java/com/starrocks/http/HttpServer.java

```diff
@@ -235,15 +245,25 @@ public void start() {
     }
 
     protected class StarrocksHttpServerInitializer extends ChannelInitializer<SocketChannel> {
+        private final Optional<SslContext> sslContext;
+
+        public StarrocksHttpServerInitializer(Optional<SslContext> sslContext) {
```

Comment:
Using Optional as parameter will only make the code complex. It's not a behavior.